app/templates/components/ui-table/cell/admin/users/cell-event-roles.hbs

```diff
@@ -1,5 +1,55 @@
-<div class="ui ordered list">
-  {{#each record.eventRoles as |eventRole|}}
-    <div class="item">{{eventRole.name}} ({{eventRole.event.name}})</div>
+{{#if record.isUserOrganizer}}
```

Comment:
This is not the right way to do this. You're basically looping through all events that user is a part of. Whether he is an organizer or not. Then you're looping through all the organizers of that event. And then you're matching the email address. So many requests and loops.

Couldn't you just use the `coorganizer-events`, `moderator-events`, `track-organizer-events`, `organizer-events`, etc relations of the user to directly get the events that the user is a part of. ? Also, make sure you request _only_ what you need. Here you don't need the full event info. You only need event identifier and event name. Then please request and use just that.